db_setup/app/__init__.py

```diff
@@ -120,8 +120,11 @@ def update_resource(resource, existing_resource):
     existing_resource.notes = resource.get('notes', '')
     existing_resource.languages = resource['languages']
 
-start = time.perf_counter()
-import_resources()
-stop = time.perf_counter()
-print('Finished populating DB from resources.yml')
-print("Elapsed time: %.1f [min]" % ((stop-start)/60))
+@app.cli.command('populate_db')
+def do_import():
+    print('Populating DB from resources.yml...')
+    start = time.perf_counter()
+    import_resources()
+    stop = time.perf_counter()
+    print('Finished populating DB from resources.yml')
+    print("Elapsed time: %.1f [min]" % ((stop-start)/60))
```

Comment:
This looks great. Sometime in the future we should remove the tuple style string formatting and go to f strings.